spec/services/search/base_spec.rb

```diff
@@ -3,6 +3,13 @@
 RSpec.describe Search::Base, type: :service do
   let(:document_id) { 123 }
 
+  def recreate_tag_index
```

Comment:
By only recreating a single index for these tests instead of all of them using `elasticsearch_reset: true` this spec's runtime went from 20+ seconds to 8 seconds.